Follow up changes to recent updates (#6927)
microsoft/vscode-docs: docs/cpp/config-clang-mac.md

    ![Hello World Terminal Output](images/clang-mac/helloworld-terminal-output.png)

> The play button has two modes: **Run C/C++ File** and **Debug C/C++ File**. The default is the last-used mode. If you see the debug icon in the play button, you can select the play button to debug, instead of selecting the drop-down menu item.

Reviewer: ```suggestion
The play button has two modes: **Run C/C++ File** and **Debug C/C++ File**. The default is the last-used mode. If you see a play button with a bug icon, you can click the play button to debug, instead of selecting the drop-down menu item.
```

Author: Prefer "select" over "click" since "select" doesn't imply a mouse (more accessible phrasing)

Author: I think I'll leave it as the "debug icon" as we might update the bug in the future

    The **Step Over** command skips over all the internal function calls within the `vector` and `string` classes that are invoked when the `msg` variable is created and initialized. Notice the change in the **Variables** window. The contents of `msg` are visible because that statement has completed.
1. Press **Step over** again to advance to the next statement (skipping over all the internal code that is executed to initialize the loop). Now, the **Variables** window shows information about the loop variable.

> 1. Press **Step over** again to execute the `cout` statement. **Note** As of the March 2019 version of the extension, no output will appear in the DEBUG CONSOLE until the last `cout` completes.

Author: Oops

> 1. Press **Step over** again to execute the `cout` statement. **Note** As of the March 2019 version of the extension, no output will appear in the DEBUG CONSOLE until the last `cout` completes.

Author: Oh, I see. The GitHub PR review shows this as a delete and add.

> 1. Press **Step over** again to execute the `cout` statement. **Note** As of the March 2019 version of the extension, no output will appear in the DEBUG CONSOLE until the last `cout` completes.

Reviewer: I meant the fact that the note "(Note that the C++ extension does not print any output to the **Debug Console** until the last cout executes.)" is preserved elsewhere but not here.

> 1. Press **Step over** again to execute the `cout` statement. **Note** As of the March 2019 version of the extension, no output will appear in the DEBUG CONSOLE until the last `cout` completes.

Author: Good pattern matching. Alexandra has updated the Clang tutorial flow to not need the note but hasn't updated the other four tutorials. She will go through the others and remove that note in a later PR.
For now, just removing the mention of 2019
